anotherfile.js

```
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
+    case "add":
+      return a + b;
+    case "subtract":
+      return a - b;
+    case "multiply":
+      return a * b;
+    case "divide":
+      if (b === 0) return "Error: Cannot divide by zero";
+      return a / b;
+    default:
+      return "Error: Unknown operation";
```

The function should consistently return a number or throw an error instead of returning a string for error cases.
Suggestion: throw new Error("Unknown operation");

**Original code:**
```
return "Error: Unknown operation";
```